DeadOrAlive.js

```diff
@@ -54,49 +56,78 @@ const getFirstHumanEntity = async (entities) => {
     return personEntity;
 };
 
-const getResultModel = (personEntity) => {
+const getWikipediaModel = (personEntity) => {
     const {
         P569: birthData,
         P570: deathData
     } = personEntity.claims;
+
     const name = personEntity.labels.en.value;
     const wikipediaUrl = parseWikipediaUrl(personEntity.sitelinks.enwiki.title);
     const hasDOB = birthData !== undefined;
     const isDead = deathData !== undefined;
 
-    let dateOfBirthString;
     let dateOfBirth;
-    let age = null;
     if (hasDOB) {
-        dateOfBirthString = birthData[0].mainsnak.datavalue.value.time;
-        dateOfBirth = moment(dateOfBirthString, WikiDataDateFormat);
-        age = moment().diff(dateOfBirth, 'years');
+        const dateOfBirthString = birthData[0].mainsnak.datavalue.value.time;
+        dateOfBirth = moment(dateOfBirthString, WikiDataDateFormat).toDate();
     }
 
-    let dateOfDeathFormatted = null;
+    let dateOfDeath;
     if (isDead) {
         const dateOfDeathString = deathData[0].mainsnak.datavalue.value.time;
-        const dateOfDeath = moment(dateOfDeathString, WikiDataDateFormat);
+        dateOfDeath = moment(dateOfDeathString, WikiDataDateFormat).toDate();
+    }
+
+    return {
+        name,
+        dateOfBirth,
+        dateOfDeath,
+        wikipediaUrl
+    };
+};
 
+const getResultModel = (wikipediaModel) => {
```

Comment:
Could destructure the params to avoid repeating `wikipediaModel` within the func. e.g.
```javascript
const getResultModel = ({ dateOfBirth, dateOfDeath, name, wikipediaUrl }) => {}
```

But then might lose that context 'this is a wikipedia model', so not a definite enhancement 🤔

Reply:
Just thinking it's not a Wikipedia model really, as it could be from any data source and the URL can point to any website. Case in point, the overrides are an alternative data source for this model. I might rename everything for more clarity. 

Thanks for the feedback.